fecgov/fec-cms
# Update IE support messaging

## Summary

- Resolves #4648 

Updated messaging to indicate that we are no longer supporting Internet Explorer.

### Required reviewers

1 front end and 1 content

## Impacted areas of the application

General components of the application that this PR will affect:

-  Anyone using Internet Explorer

## Screenshots

![Screen Shot 2021-05-26 at 12 53 20 PM](https://user-images.githubusercontent.com/12799132/119700598-699c9600-be21-11eb-811a-9d07540fa06a.png)

## How to test

- For content staff: take a look at the screenshot to see if the language in the black banner for no longer support IE looks ok.
- For front end: check out this branch and try it in Internet Explorer. Make sure this banner appears for all versions of Internet Explorer.

- [WIKI on testing local dev server on Windows](https://github.com/fecgov/fec-cms/wiki/Testing-a-dev-branch-on-iPhone,-Internet-Explorer-Windows,-Android)
